Library/Formula/hugo.rb

```diff
@@ -195,8 +175,10 @@ def install
     ln_sf buildpath, buildpath/"src/github.com/spf13/hugo"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "main.go"
-    bin.install "main" => "hugo"
+    system "go", "build", "-o", bin/"hugo", "main.go"
+
+    bash_completion.mkpath
+    system bin/"hugo", "genautocomplete", "--completionfile=#{bash_completion}/hugo.sh"
```

Comment:
For me at least, when it's in `post_install` Homebrew doesn't automatically link the completion script into `/usr/local/etc/bash_completion.d`. I have to `brew unlink hugo && brew link hugo`. Is that expected?